phpBB/styles/prosilver/theme/content.css

```diff
@@ -466,17 +466,24 @@ blockquote cite {
 	/* Username/source of quoter */
 	font-style: normal;
 	font-weight: bold;
-	margin-left: 20px;
+	margin-left: 0;
 	display: block;
 	font-size: 0.9em;
 }
 
+blockquote cite:before, .uncited:before {
+	/* Font Awesome quote-left */
+	content: '\f10d';
+	font-family: FontAwesome;
+	padding-right: 5px;
+}
+
 blockquote cite cite {
 	font-size: 1em;
 }
 
 blockquote.uncited {
-	padding-top: 25px;
+	padding-top: 0;
 }
```

Comment:
Appreciate the education. I presume this is to optimize/remove redundant information. Might I ask if you use a tool to determine this or just eye-ball it? I'm using Chrome's Inspect tool for testing.